### DISCUSSION TITLE
Support for grpc-web protocol (HTTP1.1)

### DISCUSSION THREAD

**Discussion by Pill30** [zalopay-oss/jmeter-grpc-request#284] [General]
Hi,
I can see that the ToDo list has 'Supports grpc-web protocol (HTTP1.1)' as outstanding.
Do you have a timeline (or even an alpha build) with this functionality?
The current version (1.2.6) returns "HTTP status code 464 - Incompatible protocol"

FYI: When using the utility BloomRPC, I can correctly call the gRPC method when set to grpc-web.

Any advice would be great. 
Thanks in anticipation.

**Comment by nptri** [comment]
The library discontinuing, I advise you to pick other tools such as k6
